.circleci/config.yml

```diff
@@ -494,7 +494,7 @@ job_configuration:
     resource_class_to_use: medium+
   - &config-2_4
     <<: *filters_all_branches_and_tags
-    ruby_version: '2.4'
+    ruby_version: '2.4.10'
```

Comment:
Consider perhaps documenting here why this is different from the others? Otherwise it may be a bit confusing next time someone needs to update the images :)